javascripts/main.js

```diff
@@ -20,10 +20,41 @@ define([
       : location.href + 'filters';
   };
 
+  // inspired by https://stackoverflow.com/a/6109105/1363815 until I have a better
+  // idea of what we want to do here
+  function relativeTime(current, previous) {
```

Comment:
This function is very rough, and if someone wants to polish this at all they're welcome to